course-lessons/python-fundamentals/python-fundamentals-variables.md

```diff
@@ -0,0 +1,200 @@
+    else:
+        print("Variable int_var exists, but is not an integer.\n",
+              "Make sure you assigned the value of int_var to be a whole number.")
+except NameError:
+    print("Can't find a variable named int_var, make sure you've\n",
+          "correctly spelled the variable name and assigned it a value!")
+
+try:
+    print("Variable float_var has been successfully assigned the value " +
+          str(float_var) + ".")
+    if isinstance(float_var, float):
+        print("Variable float_var is a float, good job!")
+    else:
+        print("Variable float_var exists, but is not a float.\n",
+              "Make sure you assigned the value of float_var to be a number with a decimal value.")
+except NameError:
+    print("Can't find a variable named float_var, make sure you've\n",
+          "correctly spelled the variable name and assigned it a value!")
+
+try:
+    print("Variable string_var has been successfully assigned the value " +
+          str(string_var) + ".")
+    if isinstance(string_var, str):
+        print("Variable string_var is a string, good job!")
+    else:
+        print("Variable string_var exists, but is not a string.\n",
+              "Make sure you assigned the value of string_var to be a value surrounded by quotes.")
+except NameError:
+    print("Can't find a variable named string_var, make sure you've\n",
+          "correctly spelled the variable name and assigned it a value!")
+```
+
+## Check Variable Type
+
+After you create a variable, you can check the type using the following syntax:
+
+`type(variable_name)`
+
+For example, you can check the type of `boulder_precip_in` after each time that a variable is created with that same name to see that the type changes from `int` to `float`. 
+
+```python
+boulder_precip_in = 21
+
+type(boulder_precip_in)
+```
+
+```python
+boulder_precip_in = 20.68
+
+type(boulder_precip_in)
+```
+
+You can also check the type of the variable named `city`, which is a string (`str`) regardless of how many words (or punctuation) it contains. 
+
+```python
+city = "Boulder"
+
+type(city)
+```
+
+```python
+city = "Boulder, CO is the home of the University of Colorado, Boulder campus."
+
+type(city)
+```
+
+Checking the type for a variable can help you make sure that you understand what a variable contains and how it can be used. 
+
+For example, you can actually create a `str` variable that contain numbers if you use the syntax for creating a `str` variable (e.g. `variable_name = "value"`).
+
+Notice below that `city_precip` is still a `str`, even though it contains a number.
+
+```python
+city_precip = "20.68"
+
+type(city_precip)
+```
+
+Thus, the value itself is not important for determining whether the variable is a numeric or string type - the syntax does this. 
+
+A good reminder that it is important to make sure that you are defining variables with the appropriate syntax to distinguish between numeric and string types. 
```

Comment:
Let's add an interactive activity for checking type that mirrors the interactive activity, so that they check the type of the variables that they previously created.